io.enpass.Enpass.yaml

```diff
@@ -0,0 +1,101 @@
+  - --share=network
+    # Required to save vaults in default location <$HOME/Documents>
+  - --filesystem=xdg-documents
+modules:
+  - name: enpass
+    buildsystem: simple
+    build-options:
+      env:
+        FLATPAK_ID: io.enpass.Enpass
+    build-commands:
+      - |
+
+        mkdir -p ${FLATPAK_DEST}/bin/extra
+        mkdir -p ${FLATPAK_DEST}/share/app-info/icons/flatpak/128x128
+        mkdir -p ${FLATPAK_DEST}/share/mime/packages
+
+        install -Dm 755 apply_extra --target-directory ${FLATPAK_DEST}/bin/
+        chmod +x ${FLATPAK_DEST}/bin/apply_extra
+
+        install -Dm 755 enpass --target-directory ${FLATPAK_DEST}/bin/
+        chmod +x ${FLATPAK_DEST}/bin/enpass
+
+        install -Dm 644 ${FLATPAK_ID}.desktop --target-directory ${FLATPAK_DEST}/share/applications/
+        install -Dm 644 ${FLATPAK_ID}.metainfo.xml --target-directory ${FLATPAK_DEST}/share/metainfo/
+        install -Dm 644 ${FLATPAK_ID}.mime.xml --target-directory ${FLATPAK_DEST}/share/mime/packages/
+
+        install -Dm 644 ${FLATPAK_ID}.png --target-directory ${FLATPAK_DEST}/share/app-info/icons/flatpak/128x128/
+        install -Dm 644 ${FLATPAK_ID}.svg --target-directory ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/
+    sources:
+      - type: script
+        dest-filename: enpass
+        commands:
+          - exec /app/extra/enpass/Enpass
+      - type: extra-data
+        filename: enpass.deb
+        only-arches: [x86_64]
+        url: https://apt.enpass.io/pool/main/e/enpass/enpass_6.8.5.1173_amd64.deb
+        sha256: 5855e617041d73682320f3643eb4136c93eef2beaf3be9d37cbadfc76d719b5b
+        x-checker-data:
+          type: debian-repo
+          package-name: enpass
+          root: https://apt.enpass.io
+          dist: stable
+          component: main
+        size: 39644988
+      - type: script
+        dest-filename: apply_extra
+        commands:
+          - |
+            # =====$(pwd)=====`/app/extra`=====
+            # Env from `build-options` does not carry over, define vars again
+            FLATPAK_ID=io.enpass.Enpass
+            # Extract upstream package
+            mkdir -p deb-package export/{bin,share/{icons,mime}}
+
+            bsdtar -xvf enpass.deb
+            tar -xzf data.tar.gz -C deb-package
+
+            # Install icons
+            mv deb-package/usr/share/icons export/share/
+
+            # Rename all icons accordingly
+            rename enpass ${FLATPAK_ID} export/share/icons/hicolor/*/{apps,status}/*
+
+            # Install the app
+            mv deb-package/opt/enpass ./
+      - type: file
+        path: icons/io.enpass.Enpass.png
+      - type: file
+        path: icons/io.enpass.Enpass.svg
+      - type: file
+        path: io.enpass.Enpass.desktop
+      - type: file
+        path: io.enpass.Enpass.metainfo.xml
+      - type: file
+        path: io.enpass.Enpass.mime.xml
+    cleanup:
+     - control.tar.gz
+     - data.tar.gz
+     - debian-binary
+     - deb-package
+     - enpass.deb
```

Comment:
this does nothing.

Reply:
Thank you for the clarification. I've moved the cleanup back to the `apply_extra` script.